curves/bls12_377/src/curves/g2.rs

```diff
@@ -58,15 +57,15 @@ impl<H: HostFunctions> SWCurveConfig for Config<H> {
     /// AFFINE_GENERATOR_COEFFS = (G2_GENERATOR_X, G2_GENERATOR_Y)
     const GENERATOR: G2Affine<H> = G2Affine::<H>::new_unchecked(G2_GENERATOR_X, G2_GENERATOR_Y);
 
-    #[inline(always)]
-    fn mul_by_a(_: Self::BaseField) -> Self::BaseField {
-        Self::BaseField::zero()
-    }
+    // #[inline(always)]
+    // fn mul_by_a(_: Self::BaseField) -> Self::BaseField {
+    //     Self::BaseField::zero()
+    // }
```

Comment:
@achimcc why in prev implementation you hardcoded `mul_by_a` as `ZERO`?
If the goal is  to prevent potential calls to this method maybe we should document it but allow to perform the operation anyway. In this way we are just silently returning the wrong result

Reply:
https://github.com/arkworks-rs/algebra/blob/master/ec/src/models/short_weierstrass/mod.rs#L33

It's just saying the A coefficient in zero in the curve equation.

Reply:
Missed that. Ty